src/skmultiflow/evaluation/evaluate_prequential_delayed.py

```diff
@@ -0,0 +1,518 @@
+    >>> from skmultiflow.evaluation import EvaluatePrequential
+    >>>
+    >>> # Set the stream
+    >>> stream = SEAGenerator(random_state=1)
+    >>>
+    >>> # Set the model
+    >>> ht = HoeffdingTreeClassifier()
+    >>>
+    >>> # Set the evaluator
+    >>>
+    >>> evaluator = EvaluatePrequential(max_samples=10000,
+    >>>                                 max_time=1000,
+    >>>                                 show_plot=True,
+    >>>                                 metrics=['accuracy', 'kappa'])
+    >>>
+    >>> # Run evaluation
+    >>> evaluator.evaluate(stream=stream, model=ht, model_names=['HT'])
+
+    >>> # The second example demonstrates how to compare two models
+    >>> from skmultiflow.data import SEAGenerator
+    >>> from skmultiflow.trees import HoeffdingTreeClassifier
+    >>> from skmultiflow.bayes import NaiveBayes
+    >>> from skmultiflow.evaluation import EvaluateHoldout
+    >>>
+    >>> # Set the stream
+    >>> stream = SEAGenerator(random_state=1)
+    >>>
+    >>> # Set the models
+    >>> ht = HoeffdingTreeClassifier()
+    >>> nb = NaiveBayes()
+    >>>
+    >>> evaluator = EvaluatePrequential(max_samples=10000,
+    >>>                                 max_time=1000,
+    >>>                                 show_plot=True,
+    >>>                                 metrics=['accuracy', 'kappa'])
+    >>>
+    >>> # Run evaluation
+    >>> evaluator.evaluate(stream=stream, model=[ht, nb], model_names=['HT', 'NB'])
+
+    >>> # The third example demonstrates how to evaluate one model
+    >>> # and visualize the predictions using data points.
+    >>> # Note: You can not in this case compare multiple models
+    >>> from skmultiflow.data import SEAGenerator
+    >>> from skmultiflow.trees import HoeffdingTreeClassifier
+    >>> from skmultiflow.evaluation import EvaluatePrequential
+    >>> # Set the stream
+    >>> stream = SEAGenerator(random_state=1)
+    >>> # Set the model
+    >>> ht = HoeffdingTreeClassifier()
+    >>> # Set the evaluator
+    >>> evaluator = EvaluatePrequential(max_samples=200,
+    >>>                                 n_wait=1,
+    >>>                                 pretrain_size=1,
+    >>>                                 max_time=1000,
+    >>>                                 show_plot=True,
+    >>>                                 metrics=['accuracy'],
+    >>>                                 data_points_for_classification=True)
+    >>>
+    >>> # Run evaluation
+    >>> evaluator.evaluate(stream=stream, model=ht, model_names=['HT'])
+
+    """
+
+    def __init__(self,
+                 n_wait=200,
+                 max_samples=100000,
+                 batch_size=1,
+                 pretrain_size=200,
+                 max_time=float("inf"),
+                 metrics=None,
+                 output_file=None,
+                 show_plot=False,
+                 restart_stream=True,
+                 data_points_for_classification=False):
+
+        super().__init__()
```

Comment:
Pass explicitly the parameters of the parent class. They should be then available.

Reply:
@jacobmontiel this is the same code for `evaluate_prequential.py`. Do we need to change both?

Reply:
Let's see if we can insert the delayed code into the original `EvaluatePrequential` and then we fix it there ;-)